ember/Dockerfile

```diff
@@ -5,6 +5,8 @@ COPY . /usr/share/app
 
 ARG OIDC_HOST
 
+RUN apt-get update && apt-get -y install git
```

Comment:
why?

Reply:
One of the yarn packages requires git to run

Reply:
Interesting. Which one? What is it doing with git?

Reply:
Can't seem to reproduce it consistently now. It didn't work once after removing this line, but after that it just worked.

Reply:
Ok, wanna remove it from the PR then?

Reply:
Yes, removed it

Reply:
@czosel @Yelinz This was added, because I'm not able to build the container without it. This did not change:

```
...
yarn install v1.22.5
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
error Couldn't find the binary git
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
Removing intermediate container 295960a54120
The command '/bin/sh -c yarn install && yarn build' returned a non-zero code: 1
ERROR: Service 'ember' failed to build : Build failed
```

Reply:
Interesting; it seems that yarn actually depends on git: https://github.com/yarnpkg/yarn/issues/6881
Did you consider using the `node:lts` base image instead of `node:lts-slim`? That might include git as well.